pkg/parca/parca_test.go

```diff
@@ -171,6 +171,8 @@ func MustReadAllGzip(t require.TestingT, filename string) []byte {
 }
 
 func TestConsistency(t *testing.T) {
+	t.Skipf("skipped, need to think how we want to bring back consistency exports without pprof_labels")
```

Comment:
Given we don't store the pprof_labels anymore, it's impossible to export the same profile as pprof download from now on.